python/tank/bootstrap/manager.py

```diff
@@ -120,14 +120,37 @@ def __repr__(self):
 
         repr  = "<TkManager "
         repr += " User %s\n" % self._sg_user
-        repr += " Cache fallback path %s\n" % self._bundle_cache_fallback_paths
+        repr += " Cache fallback path %s\n" % self._get_bundle_cache_fallback_paths()
```

Comment:
nitpick: i suggest we change this to `Bundle cache fallback path xxx` just to be clear which cache we are talking about.